scripts/pylint_extensions_test.py

```diff
@@ -47,15 +48,27 @@ def test_finds_non_explicit_keyword_args(self):
         checker_test_object.CHECKER_CLASS = (
             pylint_extensions.ExplicitKeywordArgsChecker)
         checker_test_object.setup_method()
-        func_call_node_one, func_call_node_two, func_call_node_three = (
-            astroid.extract_node("""
+        (
```

Comment:
Why do we require this block?

Reply:
There are a lot of variables which won't fit on the same line, that's why I used the braces